src/sentry/receivers/outbox/control.py

```diff
@@ -13,3 +13,10 @@ def process_user_updates(object_identifier: int, **kwds: Any):
     if (user := maybe_process_tombstone(User, object_identifier)) is None:
         return
     user  # Currently we do not sync any other user changes, but if we did, you can use this variable.
+
+
+@receiver(process_control_outbox, sender=OutboxCategory.INTEGRATION_UPDATE)
+def process_integration_updates(object_identifier: int, **kwds: Any):
+    if (integration := maybe_process_tombstone(Integration, object_identifier)) is None:
```

Comment:
For now, we only sync tombstones, which occurs when the integration associated with an `integration_updated` is queried but does not exist.